packages/engine/src/migration/service.ts

```diff
@@ -32,7 +32,9 @@ export class MigrationService extends Service {
     this.srcVersion = process.env.TESTMIG_DB_VERSION || this.meta.get()?.version || '0.0.0'
     this.dstVersion = process.env.MIGRATE_TARGET || this.meta.app().version
     this.autoMigrate = !!yn(process.env.AUTO_MIGRATE) || !!process.env.MIGRATE_CMD?.length
-    this.isDown = process.env.MIGRATE_CMD === 'down'
+    this.isDown = process.env.MIGRATE_CMD
+      ? process.env.MIGRATE_CMD === 'down'
+      : semver.lt(this.dstVersion, this.srcVersion)
```

Comment:
Before, the logic was: if you don't explicitly specify that you want to run a down migration, we assume you're doing an up migration. Now, if there is no migration command specified expressly by the user, we check if the destination is lower than the source version to know if we're in the case of a down migration.